linux_os/guide/system/software/integrity/software-integrity/aide/aide_check_audit_tools/bash/shared.sh

```diff
@@ -18,6 +18,12 @@
 {{% set auditfiles = auditfiles + ["/usr/sbin/audispd"] %}}
 {{% endif %}}
 
+{{% if 'rhel' not in product %}}
```

Comment:
I'm not sure how valid or expected this change is to all products.
Is your change in the context of SLE only? Or all products that are not RHEL?

If in context of SLE, suggest the following, as this keeps the `configString` the same for the rest of products.
```suggestion
{{% if 'sle' in product %}}
```